yarn-project/acir-simulator/src/client/view_data_oracle.ts

```diff
@@ -36,11 +36,11 @@ export class ViewDataOracle extends TypedOracle {
   }
 
   /**
-   * Return the secret key of a owner to use in a specific contract.
-   * @param owner - The owner of the secret key.
+   * Return the nullifier key pair of an account to use in a specific contract.
+   * @param account - The account address of the nullifier key.
    */
-  public getSecretKey(owner: PublicKey) {
-    return this.db.getSecretKey(this.contractAddress, owner);
+  public getNullifierKeyPair(account: AztecAddress) {
+    return this.db.getNullifierKeyPair(account, this.contractAddress);
```

Comment:
This should be storage contract address. Will change in another PR when the address is actually used to silo the key.